### PR TITLE
🐛 [PRTL-856] Annotation count missing

### PR DESCRIPTION
**Pull request by SidSyrus**

### CODE REVIEW

File: modules/node_modules/@ncigdc/containers/CaseTr.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ export const CaseTrComponent = ({ node, index, theme, relay, total }: TProps) =>
           {findDataCategory('Bio', node.summary.data_categories).file_count}
         </FilesLink>
       </Td>
-      <Td>annotations</Td>
+      <Td>{node.annotations.hits.total}</Td>
```
Comment (SidSyrus) [inline modules/node_modules/@ncigdc/containers/CaseTr.js]: There are no link in PROD. @phuongmy Can you input on this?

Comment (Jephuff) [inline modules/node_modules/@ncigdc/containers/CaseTr.js]: theres a link, you might be looking at dev2, make sure you aren't connected to a vpn
https://portal.gdc.cancer.gov/search/c?filters=%7B%22op%22:%22and%22,%22content%22:%5B%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22files.file_id%22,%22value%22:%5B%22cfdfa752-2059-46b6-a948-b55245897df4%22%5D%7D%7D%5D%7D&facetTab=files

Comment (SidSyrus) [inline modules/node_modules/@ncigdc/containers/CaseTr.js]: aah, Yes I was! 😓

File: modules/node_modules/@ncigdc/containers/FileTable.js [hunk 2]
```diff
@@ -159,6 +160,17 @@ export const FileTableQuery = {
                 }
               }
             }
+            annotations {
+              hits(first:20) {
+                total
+                edges {
+                  node {
+                    annotation_id
+                    entity_id
```
Comment (Jephuff) [inline modules/node_modules/@ncigdc/containers/FileTable.js]: Is `entity_id` used in the file table?

**Pull request merged**